scripts/perf-test.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -eo pipefail
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+NUM_RECORDS=${NUM_RECORDS:-10000000}
+RECORD_SIZE=${RECORD_SIZE:-1024}
+ON_SHUTDOWN=()
+PERF_TEST=${SCRIPT_DIR}/perf-tests
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NOCOLOR='\033[0m'
+
+. "${SCRIPT_DIR}/common.sh"
```

Comment:
Trying to make a run on CI, I got this error: `Unable to resolve path to kubectl`. Maybe the way `kubectl` is installed in Jenkins is not compatible with `command`. 

Anyway, looking into the `common.sh` script, there are some commands that are not supported on Jenkins, like `kustomize` or `gawk`. I wonder if this common script is really needed here only to get `docker` command.

Reply:
you are right, we probably don't need common.sh